.nvmrc

```diff
@@ -0,0 +1 @@
+v20.1.0
```

Comment:
Why?

Reply:
So you run it with `node20`, which is the version set for the runner https://github.com/zattoo/output/blob/main/action.yml#L18

Reply:
I thought we update everything to 16 not 20, Web is still using 16